ui/perfherder/FilterControls.jsx

```diff
@@ -29,6 +33,7 @@ export const createDropdowns = (dropdownOptions, colClass, outline = false) => (
             options={dropdown.options}
             selectedItem={dropdown.selectedItem}
             updateData={dropdown.updateData}
+            dropdownNamespace={dropdown.namespace}
```

Comment:
Nit: would be nice to keep the name consistent so it's easier to trace (`namespace` prop to correspond with the `dropdown.namespace`.

Reply:
Done.